includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2105,6 +2117,14 @@ protected function get_account_branding_secondary_color( $default_value = '' ):
 		return $default_value;
 	}
 
+	/**
+	 * Retrieves the country of the current account.
+	 */
+	protected function get_account_domestic_currency(): string {
+		$merchant_country = $this->account->get_account_country();
+		return $this->localization_service->get_country_locale_data( $merchant_country )['currency_code'];
```

Comment:
Technically, `get_country_locale_data` can return empty array. See doc comment
```php
@return array Array with the country's locale data. Empty array if country not found.
```

Please consider, if we need to handle this case.